src/meta-srv/src/handler/failure_handler/runner.rs

```diff
@@ -165,6 +182,10 @@ pub(crate) struct FailureDetectorEntry<'a> {
 }
 
 impl FailureDetectorEntry<'_> {
+    fn region_ident(&self) -> RegionIdent {
```

Comment:
I prefer to return `&RegionIdent` personally. The API user can choose whether to clone it.